johnsonandjohnson/Bodiless-JS
# fix(vital-cards): reorg of cards & api doc [canary]

## Changes
- Fixed the as promise error see in this screenshot: https://user-images.githubusercontent.com/58534559/172922333-e7db5797-7e5f-4d4c-9e64-48ddc419545a.jpg
- Clean up vital card variations and added the tokens:
-  WithHorizontalContentAtTop
- WithHorizontalContentCentered
- WithHorizontalLeftOrientation
- WithHorizontalRightOrientation
- Moved the CTA link styles to Base.ts
- Stubbed out Topic, Category, Product
- Defined the card categories. Type, Usage, CTA Type, CTA Style, Features 
- Add Documentation

## Test Instructions
- Verify on static there is no console errors on pages with cards. (see first bullet above)
- Retest Cards -- I would say not need to test all functionality but do some smoke testing
--- with focus on /styleguide/cards and adding cards to flowcontainer
- Comonent Picker - Cards -- run through and see if they make sense
